addons/web/static/src/core/install_prompt/install_prompt.scss

```diff
@@ -0,0 +1,30 @@
+.o_install_prompt {
+    border-radius: 1em;
+    height: unset !important;
+	-webkit-backdrop-filter: blur(5px);
+	backdrop-filter: blur(5px);
+	background: rgba(#FFF, .7);
```

Comment:
About the background's color, what happens in dark mode ? Could you use an existing variable instead (at least for the "#FFF") ?